Diagnostic/diagnostic.py

```diff
@@ -890,7 +890,7 @@ def install_omi():
         mysqladdress=readPrivateConfig("mysqladdress")
         mysqlusername=readPrivateConfig("mysqlusername")
         mysqlpassword=readPrivateConfig("mysqlpassword")
-        RunGetOutput("/opt/microsoft/mysql-cimprov/bin/mycimprovauth default "+mysqladdress+" "+mysqlusername+" '"+mysqlpassword+"'")
+        RunGetOutput("/opt/microsoft/mysql-cimprov/bin/mycimprovauth default "+mysqladdress+" "+mysqlusername+" '"+mysqlpassword+"'", log_cmd=False)
```

Comment:
Unfortunately, this doesn't look right on LAD, as LAD defined a new function LogRunGetOutput, and assign LogRunGetOutput to RunGetOutput.

It looks like we need to change the param name here to 'should_log', and also in LogRunGetOutput(), the actual waagent.RunGetOutput() call should be 'waagent.RunGetOutput(log_cmd=should_log)'... Please take a look at lines 63-72.

Thanks much for doing this!